.gitignore

```diff
@@ -3,3 +3,6 @@
 
 # Ada Library Information
 *.ali
+# .gitignore
+node_modules/
+.md 
```

Comment:
- [ ] please try to add the `test.md` to `.gitignore` file  instead of `.md` as required and also try to remove the `test.md`

Reply:
dear @youmari, I have added test.md to gitignore